zingolib/src/lib.rs

```diff
@@ -6,6 +6,7 @@ mod test_framework;
 pub mod blaze;
 pub mod commands;
 pub mod compact_formats;
+pub mod error;
```

Comment:
@AloeareV @fluidvanadium why not make `error` into a crate?

`zingo-error`?

Reply:
each crate should have its own error enum